playwright.config.ts

```diff
@@ -0,0 +1,108 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
```

Comment:
My preference is for `playwright.config.ts` to exist in the test/e2e folder, but if it must be at the repo root then I'll be fine.

Reply:
There *should*, as far as I can tell, be a way for the config file to live in the `test/e2e` folder, but when I tried I was getting strange errors like "apiRequestContext.post: Invalid URL" in unrelated parts of the code. It looks like moving this file will be a slightly bigger project than I anticipated it would be, so I think I'll defer that to a later PR.